src/sonic-yang-models/tests/yang_model_pytests/test_dash_crm.py

```diff
@@ -0,0 +1,105 @@
+import pytest
+from copy import deepcopy
+
+
+base_data = {
+    "sonic-device_metadata:sonic-device_metadata": {
+        "sonic-device_metadata:DEVICE_METADATA": {
+            "localhost": {
+                "switch_type": "dpu"
+            }
+        }
+    }
+}
+
+
+@pytest.fixture
+def data(request):
```

Comment:
why do you need this fixture?

Reply:
It is needed to pass a base "sonic-device_metadata" configuration into each test without copying it.